results-2021.md

```diff
@@ -0,0 +1,461 @@
+| GOV.UK Design System Community resource | 10 | 18.18% |
+| GOV.UK Frontend Toolkit | 10 | 18.18% |
+| GOV.UK Elements | 5 | 9.09% |
+| GOV.UK Template | 5 | 9.09% |
+| NHS Design System (NHS Frontend) | 4 | 7.27% |
+| govuk-react | 2 | 3.64% |
+| HMRC Design System (HMRC Frontend) | 2 | 3.64% |
+| MOJ Pattern Library (an extension of the GOV.UK Design System) | 2 | 3.64% |
+| GOVUK publishing components | 1 | 1.82% |
+| Hackney Design System | 1 | 1.82% |
+| None | 4 | 7.27% |
+| Don't know | 2 | 3.64% |
+
+## Question 12: How are these frontend resources integrated into your project?
+
+### Answers
+
+Answer are categorised
+
+| Name | Count | Percentage |
+| --- | --- | --- |
+| npm | 22 | 40.00% |
+| Manually | 1 | 1.82% |
+| Gulp | 1 | 1.82% |
+| We don't | 1 | 1.82% |
+| Departmental resource | 3 | 5.45% |
+| Gem | 2 | 3.64% |
+| Webpack | 2 | 3.64% |
+| Other | 2 | 3.64% |
+| Package manager | 1 | 1.82% |
+| NuGet | 1 | 1.82% |
+| No answer | 19 | 34.55% |
+
+## Question 13: How do you keep these resources up-to-date in your project?
+
+### Answers
+
+Answer are categorised
+
+| Name | Count | Percentage |
+| --- | --- | --- |
+| npm | 11 | 20.00% |
+| Dependency notification service (e.g Dependabot) | 9 | 16.36% |
+| CI/CD pipeline | 5 | 9.09% |
+| Manually | 4 | 7.27% |
+| Watch for changes | 2 | 3.64% |
+| Rarely | 2 | 3.64% |
+| Bundler | 1 | 1.82% |
+| With difficulty | 1 | 1.82% |
+| Not sure | 1 | 1.82% |
+| No answer | 19 | 34.55% |
+
+## Question 14: Does your project use any automated accessibility or performance testing tools? List any tools that you find useful in your project.
+
+### Answers
+
+| Name | Count | Percentage |
+| --- | --- | --- |
+| aXe | 18 | 32.73% |
+| Lighthouse | 8 | 14.55% |
+| Cypress | 4 | 7.27% |
+| Wave | 3 | 5.45% |
+| Cucumber | 3 | 5.45% |
+| Speed Curve | 3 | 5.45% |
+| SiteImprove | 1 | 1.82% |
+| Firefox Accessibility Inspector | 1 | 1.82% |
+| Webpack bundle checkers | 1 | 1.82% |
+| JavaScript tests | 1 | 1.82% |
+| Not sure | 2 | 3.64% |
+| No | 9 | 16.36% |
+| No Answer | 17 | 30.91% |
+
+## Question 15: What CSS pre-processor do you use on your project?
+
+### Answers
+
+| Name | Count | Percentage |
+| --- | --- | --- |
+| Sass | 40 | 72.73% |
+| We don't use a pre-processor | 4 | 7.27% |
+| Less | 2 | 3.64% |
+| PostCSS | 1 | 1.82% |
+| SCSS | 1 | 1.82% |
+| We don't write our own styles | 1 | 1.82% |
+| I do not know | 7 | 12.73% |
+
+## Question 16: What SASS compiler do you use on your project?
+
+### Answers
+
+| Name | Count | Percentage |
+| --- | --- | --- |
+| Dart Sass | 12 | 21.82% |
+| LibSass (including node-sass) | 7 | 12.73% |
+| Ruby Sass | 4 | 7.27% |
+| gulp-sass | 1 | 1.82% |
+| Multiple | 1 | 1.82% |
+| I do not know | 12 | 21.82% |
+| No answer | 18 | 32.73% |
+
+## Question 17 & 18: Ruby Sass and Lib Sass (including node-sass) are deprecated. Do you have any plans to migrate to Dart Sass? Can you explain why you haven't made any plans to migrate to Dart Sass?
```

Comment:
[GOV.UK style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style) recommends using 'and' instead of ampersand, can we make that change here?